src/main/java/BudgetTracker.java

```diff
@@ -45,7 +46,7 @@ private void writeUserInputToDatabase() throws SQLException, ClassNotFoundExcept
     }
 
     private Opportunity createNewOpportunity() {
-        return new Opportunity(name(), description(), proposedCost(), userName(), stage());
+        return new Opportunity(name(), description(), proposedCost(), userName(), stage(), uuid());
     }
 
     private String userName(){
```

Comment:
I prefer methods that have verbs at the start of this. It looks like it's displaying a prompt to get the username, and asking user for input for it here. Is there a name that could capture this better than `userName()`?